src/utils/PlaygroundApp.js

```diff
@@ -0,0 +1,132 @@
+import { FSHTank, RawFSH } from 'fsh-sushi/dist/import';
+import { exportFHIR } from 'fsh-sushi/dist/export';
+import { logger, Type } from 'fsh-sushi/dist/utils';
+import { FHIRDefinitions } from 'fsh-sushi/dist/fhirdefs';
+import { Configuration } from 'fsh-sushi/dist/fshtypes';
+import { loadExternalDependenciesPlayground, fillTank, readConfigPlayground } from './Processing';
+
+export async function playgroundApp(input) {
+  // Hard Code config
+  let config = Configuration;
+  try {
+    config = readConfigPlayground();
+  } catch {
+    console.log('Something went wrong when creating the configuration');
```

Comment:
I think for consistency, we might want to use the `logger` here and in the few other `console.log`s you added here! I think in the cases that we are logging messages about errors, you can use `logger.error`.